client/state/data-layer/wpcom-http/actions.js

```diff
@@ -35,7 +37,11 @@ export const http = ( {
 	body,
 	method,
 	path,
-	query: method === 'GET' ? { ...query, apiVersion } : { apiVersion },
+	query: Object.assign(
+		{ apiVersion },
+		query,
+		apiNamespace && { apiNamespace },
```

Comment:
note that this will break requests which provide an `apiNamespace` as they will fail if both values are present :smile: